src/ui/src/components/AdminButtons/PauseResumeButton.vue

```diff
@@ -68,6 +83,19 @@ watchEffect(() => {
             isPaused.value = false
         }
     }
+    else if (props.wf_run_id) {
+        if (props.wf_status == "running" || props.wf_status == "in_progress" || props.wf_status == "paused" || props.can_Pause == false) {
+            allowButton.value = true
+        } else {
+            allowButton.value = false
+        }
+
+        if (props.wf_status == "paused") {
+            isPaused.value = true
+        } else {
+            isPaused.value = false
+        }
```

Comment:
This isn't something worth changing, but in general, these kind of expressions can be more succinctly expressed as:
```
isPaused.value = (props.wf_status == "paused")
```